openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -116,7 +116,7 @@ def get_invalid(cls, instance):
         prefix = prefix.replace(
             "{aov_separator}", instance.data.get("aovSeparator", "_"))
 
-        required_prefix = "maya/<scene>"
+        required_prefix = "<scene>"
```

Comment:
Isn't it odd that we're still hardcoding the beginning part of the file prefix as a not customizable requirement. I'd say we can remove this completely and rely on the project settings for image prefixes per renderer.

Thoughts?

Reply:
Solid. Let me know if I should remove that `required_prefix` validation in this PR.

Reply:
Required prefix gets removed in this PR: https://github.com/pypeclub/OpenPype/pull/3880